testsuite/MDAnalysisTests/topology/test_tprparser.py

```diff
@@ -233,3 +233,5 @@ def bonds_water(request):
 def test_settle(bonds_water):
     # There are 101 water molecule with 2 bonds each
     assert len(bonds_water) == 202
+    # The last index corresponds to the last water atom
```

Comment:
Does this test a new TPR file?

Reply:
Yes it does. It tests almost all versions of gromacs from 4.0.0 to 2018.